DNN Platform/DotNetNuke.Abstractions/Logging/ILogProperties.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// add xml docs.
+    /// </summary>
+    public interface ILogProperties : ICollection<ILogDetailInfo>, IList, ICollection, IEnumerable
```

Comment:
`IEnumerable` is already part of `ICollection<T>`, so it's fine to include it or not, but it doesn't change anything.  `IList` seems too restrictive to me (on top of `ICollection` methods it also requires access by index).  And, if we're already implementing `ICollection<T>`, I'd prefer to avoid encouraging non-generic usages if we can.

```suggestion
    public interface ILogProperties : ICollection<ILogDetailInfo>
```

Reply:
This makes sense to me, I grabbed the non-generics as they come from `ArrayList`. I'll make the change

Reply:
This has been updated in the latest code push, thanks for the feedback